src/foremast/consts.py

```diff
@@ -197,10 +197,15 @@ def _generate_security_groups(config_key):
 DOMAIN = validate_key_values(CONFIG, 'base', 'domain', default='example.com')
 ENVS = set(validate_key_values(CONFIG, 'base', 'envs', default='').split(','))
 REGIONS = set(validate_key_values(CONFIG, 'base', 'regions', default='').split(','))
+MANUAL_PIPELINE_SYNONYMS = set(
```

Comment:
I don't like SYNONYMS. 😄 

Can we just make this a list of `MANUAL_PIPELINES`

Include the name "manual" in the default list.

Reply:
Call this `ALLOWED_MANUAL_TYPES`